modules/EnsEMBL/Web/File/Utils/URL.pm

```diff
@@ -255,7 +255,7 @@ sub read_file {
       $content = $args->{'size_limit'} ? $_content : $response->{'content'};
     }
     else {
-      $error = _get_http_tiny_error($response);
+      $error = _get_http_tiny_error($response)." ($url)";
```

Comment:
Please add spaces around the concatenation operator, for improved readability.

Reply:
Done and pushed.